### DISCUSSION TITLE
Google Summer of Code 2023

### DISCUSSION THREAD

**Discussion by ssheorey** [isl-org/Open3D#5896] [GSoC]
Hello everyone,

Open3D has applied for Google Summer of Code for 2023. Check out our information and ideas page:

https://github.com/isl-org/Open3D/wiki/Google-Summer-of-Code-2023

Here is the Open3D page on GSoC's website:

https://summerofcode.withgoogle.com/programs/2023/organizations/open3d-team

If you have any questions, go ahead and ask them here.

--
Open3D team.

**Comment by Dhruvaggarwal7** [comment]
hi ! I am currently working on deep learning and neural nets using tensorflow. I am quite new to open source but done quite a few projects in deep learning . pls guide me how can I can contribute ?

**Comment by Tan141101** [comment]
Hello I am Tanmay and have been working on CNN's and different frameworks as well for computer vision tasks, would love to join as a contributor to your organization and also would love to start with some work right now, If you see me fit for any task please let me know, I will surely do my level best to complete the task with utmost accuracy.
Regards,

**Comment by Aman123lug** [comment]
Hello, I am Aman I have very good experience in Deep Learning RNNs, GRU, CNNs, LSTM, also have experience in NLP, and Machine Learning I love to contribute in to your Organization. Open Source is essential for me

**Comment by aneeshc12** [comment]
Hello! I'm an undergrad robotics researcher with a graphics background and I use open3d almost every day, so seeing o3d on the GSoC list is amazing! Both projects listed so far seem great, but I had a few questions about each of them.

About the geometry processing algorithms, I see that o3d docs have pages and functions for surface reconstruction. Are we meant to reimplement/improve them? Also, which operations do you mean by lineset functions? 

The Pixar USD project seems fun and in the USD docs, they mention that prims are organized as a tree and allow attributes to be added to them. Are we supposed to be able to do these things in o3d when creating a USD as well? There is a lot of functionality USD provides (asset path resolvers, composing and overriding prims), how much of this is to be supported?

While I look at the o3d source code and try my hand at some open issues, I'd also love it if you could point me to any other resources you think might be helpful!  
I hope to hear from you soon, thanks!

**Comment by ssheorey** [comment]
Thanks for your interest! Open3D was not selected this year, but please do look around, use it in your projects and contribute back to the community.
